index.js

```diff
@@ -70,6 +70,8 @@ app.post('/api/userDeactivation', userApi.userDeactivate);
 
 app.post('/api/userCreation', userApi.userPost);
 
+app.get('/api/verifyUserSession', authApi.check, userApi.userVerifySession);
```

Comment:
`userApi.userVerifySession` -> `userApi.verifyUserSession`
or you can do this when we refactor the api